mkdocs.yml

```diff
@@ -0,0 +1,52 @@
+site_name: starlite-jwt
+repo_url: https://github.com/starlite-jwt/starlite-jwt
```

Comment:
we need a documentation url as well, same stuff as in the starlite config.

Reply:
Yep added.